doc/integrator/extend_application.rst

```diff
@@ -55,4 +55,55 @@ Will be filled later.
 Extend the geoportal image
 --------------------------
 
-Will be filled later.
+If you need to configure your own authentication you will need to extend the ``geoportal`` Docker image.
+
+For that you will need to create a new folder named ``geoportal_custom``.
+
+An this folder add a file named ``authentication.py`` with the content you needs, the original content is:
```

Comment:
An this folder add -> In this folder, add

the content you needs -> the content you need